### PR TITLE
Add helper Makefile for more easier integration tests

### PR DESCRIPTION
**Pull request by phobologic**

### CODE REVIEW

**Comment by russellballestrini** [comment]
Looks like you have failing functional tests.

File: tests/Makefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+permissions:
+	./stacker.yaml.sh | stacker build -
+
+test: permissions
```
Comment (ejholmes) [inline tests/Makefile]: We already have a `test-functional` target in the top level Makefile. Maybe move this there? https://github.com/remind101/stacker/blob/master/Makefile#L13

Comment (ejholmes) [reply tests/Makefile]: Yeah

**Comment by phobologic** [comment]
I'm going to go ahead and merge this - we can figure out if we want to somehow merge this with the main Makefile tests later.  One thing to keep in mind - this changes the way the tests are run now quite a bit, making it easier to run a single test.  Please see the readme @ejholmes @russellballestrini

**Pull request merged**